docs/product-assembly_dm-on-openshift/src/main/asciidoc/creating-secrets-central-proc.adoc

```diff
@@ -0,0 +1,30 @@
+[id='creating-secrets-central-proc']
```

Comment:
I'll mention this only here, but it applies to basically all of your file names and anchor IDs. You've used roughly the right naming convention, but if you [look again](http://ccs-jenkins.gsslab.brq.redhat.com:8080/view/Topic%20branches/job/doc-Red_Hat_JBoss_BxMS_Contribution_Guide-branch-shared-repo/lastSuccessfulBuild/artifact/index.html#_file_names_and_anchor_names), you'll notice that the core subject of a module (e.g., secrets) is first, then any sub-proc type (e.g., create or creating), then the module type (e.g., proc). So instead of `creating-secrets-central-proc`, it would be `secrets-central-create-proc`. The reason is that in the end, all module files in the folder will be listed by subject matter rather than the actions, etc. This is particularly helpful when you have several files about secrets, say two for creating them in different ways (as you have), one for editing or updating them, and one for deleting or removing or whatever. With our convention, all of the secrets files would be together. If we did it the way you've done it, there would be a couple of "creating" ones listed together (with other non-related "creating" files), then down the line the "editing" and "deleting" ones, etc.

To see more what I mean, go to the product-user-guide directory and look at my guided-decision-tables files (or decision-tables). This is the effect that we want. Not just for ordering and appearance (which is very helpful) but also to help if/when files get renamed slightly, in which case we won't have to touch the file names or anchor IDs.

If you feel like this is strange or cumbersome, you're right :) Also, just fyi, the way you've done it here is precisely how our Mod Doc peers in other teams have opted to go, despite our discussions. I'm the one who provided the sections of that guide on file/anchor naming and on reusing modules, and they accepted all but the actual file/anchor naming format, which they declined for reasons that most of us on BXMS do not agree with. So you have followed exactly what the [Mod Doc Ref Guide](https://redhat-documentation.github.io/modular-docs/#anchor-and-file-names) requires. But recall that this is the only part of that doc that we mentioned that we deviate from, for these and other reasons.

Such subtleties and slight deviations are far from ideal. One negative side effect of such an open organization :)